test/e2e/tests/tokens/custom-token-add-approve.spec.js

```diff
@@ -135,47 +133,31 @@ describe('Create token, approve token and approve token without gas', function (
             /0x095ea7b30000000000000000000000009bc5baf874d2da8d216ae9f137804184ee5afef4/u,
           ),
         );
-        await driver.clickElement({ text: 'Next', tag: 'button' });
-
-        await driver.findElement({
-          text: 'Spending cap request for your ',
-          css: '.box--flex-direction-row',
-        });
 
-        const defaultSpendingCap = await driver.findElement({
+        await driver.clickElement({ text: 'Next', tag: 'button' });
+        // Spending cap modal is opened
+        await driver.waitForSelector({
           text: '7 TST',
           css: '.mm-box > h6',
         });
 
-        assert.equal(
-          await defaultSpendingCap.getText(),
-          '7 TST',
-          'Default value is not correctly set',
-        );
```

Comment:
Same question as above. Is this check intended to be deleted?

Reply:
Yes, we are selecting element by text then wait for text, imo it is unnecessary to conduct, let me know ur thoughts !